vault/logical_system.go

```diff
@@ -223,11 +223,10 @@ func (b *SystemBackend) rawPaths() []*framework.Path {
 // prefix. Conceptually it is similar to procfs on Linux.
 type SystemBackend struct {
 	*framework.Backend
-	Core      *Core
-	db        *memdb.MemDB
-	mfaLock   *sync.RWMutex
-	mfaLogger log.Logger
-	logger    log.Logger
+	Core       *Core
+	db         *memdb.MemDB
+	logger     log.Logger
+	mfaBackend *PolicyMFABackend
```

Comment:
Should this be something generic like MFABackend?

Reply:
@ncabatoff did the split between `PolicyMFABackend` and `LoginMFABackend`, but I _think_ this is correct to stay as it is, because the existing enterprise MFA functionality that uses the system backend is policy MFA.

Reply:
Well in OSS, we don't support `PolicyMFABackend`. Keeping this as is would not resolve in OSS as `PolicyMFABackend` is defined policy_mfa_ent.go right now.

Reply:
Ah yes, of course.

Reply:
We could probably setup something similar to how the `Core` struct hides enterprise specific fields. https://github.com/hashicorp/vault-enterprise/blob/main/vault/core.go#L199